lec12-datasets.Rmd

```diff
@@ -0,0 +1,159 @@
+
+## Team dynamics
+
+### Basics of succeeding with a team
+
+Final assignment is in a group and to succeed, you need to understand team dynamics:
+
+- Everyone is different, every team is different
+- Communication is vital to work together and to achieve the goal
+- Teams go through various stages
+- Need consensus for group norms, goals, duties/responsibilities, and conduct/behaviour
+    - Important that everyone has a stake in the project
+- Rotate roles (specifically for the leader/facilitator)
+
+### Stages of group formation
+
+- "Forming-Storming-Norming-Performing-Adjourning Model"[^group_forming]
+- Essentially, groups go through stages (not always all and not always in order):
+    1. Getting to know one other
+    2. Feeling comfortable and safer, testing boundaries
+    3. Opening up, voicing opinions, potential for conflict
+    4. Stronger bonds form, cooperative
+    5. Greater focus and energy on completion of project
+    6. Team ends after project is successfully completed
+- These stages can be fast-tracked by discussing norms, duties, and conduct early on
+
+[^group_forming]: See [Principles of Group Dynamics](https://open.lib.umn.edu/principlesmanagement/chapter/13-3-group-dynamics/) 
+
+### Roles and responsibilities
+
+- Leader/Facilitator's duty:
+    - Goal is to keep things running smoothly, focused on the task, and on track for time
+    - Keep everyone on topic and on task; stay aware of the time
+    - (Try to) mediate or resolve any conflicts (there will *always* be some type of conflict; how it's dealt with is what matters)
+    - (Try to) encourage everyone to participate and allow everyone a chance at talking
+- Recorder's duty:
+    - Goal is to write/type down main or important points raised or discussed when team is meeting
+    - Keep notes and files organized and orderly
+- Organizer's duty:
+    - Arrange for next meeting time and location
+    - Send reminders to members a day or two before meeting
+
+### Code of conduct
+
+- Vital to establishing boundaries and expectations of being a team member
+    - How do you want each member to treat each other?
+    - How do you deal with conflict?
+    - What is acceptable and unacceptable behaviour?
+- These are outlined in the code of conduct
+- Mostly its common sense (be kind and respectful)
+    - But its important that you as a team write out what everyone wants and agrees to
+
+## Exercise
+
+### As a group, complete these tasks
+
+- Get into your assigned groups
+- Introduce each other:
+    - Find out everyone's name and year of study
+    - Find out other things about each other (e.g. any plans for next year, etc)
+- Assign roles to each person (these roles will be rotated in every group setting):
+    - You need: facilitator, recorder, organizer
+    - Discuss how and when roles will be rotated *(record it)*
+- Discuss and brainstorm some codes of conduct you want your team to follow *(record it)*
```

Comment:
Ok nice, this is what I wanted with my comment above. Record it, sign it, and then unite behind it and proudly show it to us.